html2asketch/base.js

```diff
@@ -16,6 +14,9 @@ class Base {
   getID() {
     return this._objectID;
   }
+  setID(_objectID) {
```

Comment:
nit: missing blank line before method (interesting that linter didn't pick that up?)

Reply:
ok